ui/packages/consul-ui/app/templates/dc/services/show/topology.hbs

```diff
@@ -22,43 +22,93 @@ as |route|>
         </BlockSlot>
       </EmptyState>
     {{else}}
+
+
+    {{#if (or (and topology.DefaultAllow topology.FilteredByACLs topology.undefinedIntention) (and topology.WildcardIntention topology.FilteredByACLs topology.undefinedIntention))  }}
```

Comment:
🙈 The conditional works but I had to duplicate the code in the `{{else}}`. LMK if you can think of a better way to format this.

Reply:
Theres an option here to maybe make a computedProperty on the topology model

Reply:
The 2nd option does not take into account that I have to have >2 notices to show that button. That's what I'm trying to capture in the conditionals I wrote. Grouping them all together in the `or` does not guarantee that use case.

Reply:
Ahhh, of course yeah I forgot about that 🤔

Reply:
Wonder if you could do something with `compact` or `array_some` 🤔

Reply:
Maybe this?

```hbs
      {{#let (without false (array true true false false)) as |visibleNotices|}}
        {{if gt visibleNotices.length 2}}
          {{!-- make collapsable --}}
        {{/if}}
      {{/let}}

```

Reply:
I've already moved the logic to a computed func in the model. Does that work? 
I also just pushed renaming `undefinedIntention` to be `notDefinedIntention`.